### PR TITLE
Updated my profile

### PR DESCRIPTION
**Pull request by samyabrata-maji**
## Changes proposed
Updated my profile `samybrata-maji.json`

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2785"></a>

### CODE REVIEW

File: data/samyabrata-maji.json [hunk 1]
```diff
@@ -1,28 +1,49 @@
...
     {
-      "name": "LinkedIn",
+      "name": "Connect with me on LinkedIn",
       "url": "https://linkedin.com/in/samyabrata-maji",
       "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/sammaji15",
+      "icon": "FaLinkedin"
```
Comment (Panquesito7) [inline data/samyabrata-maji.json:46]: Not sure if you wanted to use this icon.
```suggestion
      "icon": "FaInstagram"
```

Comment (samyabrata-maji) [reply data/samyabrata-maji.json]: My Mistake

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**